src/dotnetTestExplorer.ts

```diff
@@ -111,28 +111,43 @@ export class DotnetTestExplorer implements TreeDataProvider<TestNode> {
         }
     }
 
+    private enableBuild(): string {
+        const dotnetBuildOption = Utility.getConfiguration().get<boolean>("build");
+        return dotnetBuildOption ? "" : "--no-build";
+    }
+
+    private enableRestore(): string {
+        const dotnetRestoreOption = Utility.getConfiguration().get<boolean>("restore");
+        return dotnetRestoreOption ? "" : "--no-restore";
+    }
+
+    private loadExclusion(name: string, value: string): string {
+        const exclusion = Utility.getConfiguration().get<string>(name);
+        return exclusion ? exclusion : value;
+    }
+
     private loadTestStrings(): Thenable<string[]> {
         this.evaluateTestDirectory();
 
-        let msBuildRootTestMsg = Utility.getConfiguration().get<string>("msbuildRootTestMsg");
-        msBuildRootTestMsg = msBuildRootTestMsg ? msBuildRootTestMsg : "The following Tests are available:";
+        const exclusions = [];
+
+        exclusions.push(this.loadExclusion("msbuildRootTestxUnitPrefix", "[xUnit.net"));

```

Comment:
Could you let me know how to run mutli test projects at the same time? In previous commit, you use `lastIndexOf(msBuildRootTestMsg)`, that is much clean. Why do you change to this now?